app/src/main/java/org/mozilla/fenix/compose/Chip.kt

```diff
@@ -0,0 +1,202 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose
+
+import android.content.res.Configuration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.ChipDefaults.chipColors
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Default layout for a clickable chip.
+ *
+ * @param text [String] displayed in this chip.
+ * @param modifier [Modifier] used to be applied to the layout of the chip.
+ * @param textColor Optional text [Color] for the chip.
+ * @param backgroundColor Optional background [Color] for the chip.
+ * @param enabled Optional [Boolean] used to determine if chip responds to user input.
+ * @param onClick Optional Callback for when the user taps this chip.
+ */
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+private fun Chip(
+    text: String,
+    modifier: Modifier = Modifier,
+    textColor: Color = FirefoxTheme.colors.textActionPrimary,
+    backgroundColor: Color = FirefoxTheme.colors.actionPrimary,
+    enabled: Boolean = true,
```

Comment:
Since we don't actually have a disabled state in the design system, I think it would make sense to remove this use case to make our components more opinionated and aligned with the system.

Reply:
removed